internal/utils/templates/init_config.toml

```diff
@@ -110,6 +110,11 @@ template = "Your code is {{ `{{ .Code }}` }} ."
 [auth.sms.test_otp]
 # 4152127777 = "123456"
 
+[auth.hook.custom_access_token]
```

Comment:
Can we also add a comment describing this field?

Reply:
Good catch, thanks!